project.clj

```diff
@@ -61,6 +61,7 @@
                  [puppetlabs/jruby-utils "0.7.0"]
                  [puppetlabs/trapperkeeper]
                  [puppetlabs/trapperkeeper-authorization]
+                 [puppetlabs/trapperkeeper-metrics "0.6.0"]
```

Comment:
We don't have to hold up the PR for this change, but might be nice to bump this version in clj-parent and update to a new clj-parent from puppetserver.  Nice to avoid pinning common version dependencies when we can.